app/models/target.rb

```diff
@@ -1,4 +1,5 @@
 class Target < ApplicationRecord
-  validates :topic, :length, :title, :latitude, :longitude, presence: true
+  validates :topic, :length, :title, :latitude, :longitude, :user_id, presence: true
```

Comment:
Use `user` instead of `user_id`, in addition to validate the presence of the id itself, it also checks that the id is from a valid user.

Reply:
Sorry, actually, in Rails 5 the presence validation is done by default for every `belong_to` association, so you don't need to add it here.